built_in_tasks/target_capture_task_xt.py

```diff
@@ -1007,41 +1014,84 @@ def update_report_stats(self): #add holds completed metric to report stats
         self.reportstats['Pseudo Reward'] = self.pseudo_reward + self.reward_count
 
     ### State Functions ###
+    def _start_wait(self):
+        super()._start_wait()
+        self.ready_played = False
+        self.set_played = False
+        self.go_played = False
+        
     def _start_prepbuff(self):
         self.sync_event('CUE') #integer code 112
-        self.ready_set_player.play() 
+        self.prep_start_time = self.get_time()
+        self.ready_tone.play()
+        self.ready_played = True
+
+    def _cycle(self):
+        super()._cycle()
+
+        if not self.set_played and self.ready_played and (self.get_time() - self.prep_start_time) >= self.tone_space:
+            self.set_tone.play()
+            self.set_played = True
+            self.sync_event('CUE', 1) #integer code 113
```

Comment:
there's a potential bug if someone uses delay of near the tone spacing: this event will happen on the same frame as the delay period starts. i think we can avoid that by making sure the delay period is not within the range of the tone spacing. i used tone_spacing=0.5 and delay=0.49 and i got an error:
```
Warning: Cannot sync more than 1 event per cycle
Overwriting [b'TARGET_OFF'] with CUE event
```

Reply:
oops that must have been a different error around the go cue; i get this error consistently when delay=0.42:
```
Warning: Cannot sync more than 1 event per cycle
Overwriting [b'TARGET_ON'] with CUE event
```